pythainlp/translate/core.py

```diff
@@ -7,6 +7,7 @@ class Translate:
 
     :param str src_lang: source language
     :param str target_lang: target language
+    :param bool use_gpu: load model to gpu
```

Comment:
I think you should add default. example: `(Default is False)`

Reply:
Okay, added to all of them